packages/ses/scripts/bundle.js

```diff
@@ -15,10 +16,11 @@ const write = async (target, content) => {
 };
 
 const main = async () => {
-  const bundle = await makeBundle(
+  const version = packageJson.version;
+  const bundle = `// v${version}\n${await makeBundle(
```

Comment:
or

```suggestion
  const bundle = `// ses@${version}\n${await makeBundle(
```

Reply:
These are both good options. I have a slight preference for matching the git/npm tag.